.github/workflows/main.yml

```diff
@@ -34,21 +37,3 @@ jobs:
 
     - name: Run tests
       run: coverage run --source=vwo setup.py test
-
-    - name: Upload coverage to Codecov
-      uses: codecov/codecov-action@v3
-      env:
-        CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
-      with:
-        fail_ci_if_error: true
-
-    - name: Notification
-      if: always()
-      id: slack
-      uses: wingify/slack-github-action@v1.15.1-wingify
-      with:
-        channel-id: 'fs-review-team'
-        slack-message: "<!here> Test on *Python-${{ matrix.python-version }}* and *${{ matrix.os }}* got *${{job.status}}* ${{job.status == 'success' && ':heavy_check_mark:' || ':x:'}} \nCommit: `${{github.event.head_commit.message}}`. \nCheck the latest build: https://github.com/wingify/vwo-python-sdk/actions"
-        color: "${{job.status == 'success' && '#00FF00' || '#FF0000'}}"
-      env:
-        SLACK_BOT_TOKEN: ${{ secrets.SLACK_NOTIFICATIONS_BOT_TOKEN }}
```

Comment:
The removals here are so that I could run this on my fork: https://github.com/bhrutledge/vwo-python-sdk/pull/1